mathics/builtin/base.py

```diff
@@ -103,7 +97,7 @@ def split_name(name: str) -> str:
     return result.lower()
 
 
-mathics_to_python = {}
+mathics_to_python = {}  # here we have: name -> string
```

Comment:
This appeared also in `mathics.builtins` . That use was removed and replaced by an import to this use.